Shopify/cli
# Allow CORS for checking if GraphiQL is running

### WHY are these changes introduced?

Shopify.dev wants to verify CLI's GraphiQL server is running so we can forward code examples from our LLM assistant.

CORS policies currently restrict us from pinging the GraphiQL server.



### WHAT is this pull request doing?



This PR loosens the CORS policy for `/graphiql/ping`

### How to test your changes?



1. Run `shopify app dev` inside a Shopify app (with this code checked out)
2. Open shopify.dev in a web browser
3. Open the developer tools console and run `fetch('http://localhost:3457/graphiql/ping').then(console.log)`
4. See a response object come back with `ok === true`

### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
